internal/website/content/howto/docstore/queries.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Queries"
```

Comment:
Similar to the suggestion on part 2, consider "Running queries" as a verb phrase title.

Reply:
done